config/githash.js

```diff
@@ -1,11 +1,15 @@
-const shell = require('shelljs')
-const fs = require('fs')
-const path = require('path')
-const filePath = path.join(__dirname, '../client/version/version.json')
-const gitCommit = shell.exec('git rev-parse HEAD', {silent: true}).stdout
-
-const gitFileContents = `{"gitCommit": "${gitCommit.replace("\n", '')}"}` // eslint-disable-line 
-fs.writeFile(filePath, gitFileContents, (err) => {
-  if (err) throw err
-  console.log(`Successfully wrote Git hash - ${gitCommit}`)
-})
+const { exec } = require('shelljs');
+const { existsSync, readFileSync, writeFileSync } = require('fs');
+const { join } = require('path');
+
+const input = join(__dirname, '../BUILD');
+const output = join(__dirname, '../client/version/version.json');
+
+let gitCommit = existsSync(input)
+  ? readFileSync(input, { encoding: 'utf-8' })
+  : exec('git rev-parse HEAD', { silent: true }).stdout;
```

Comment:
should this be `shell.exec`

Reply:
Just exec, it's imported as `const { exec } = require('shelljs');`

(eslint was complaining that `const exec = require('shelljs').exec` should prefer destructuring)